playground/backend/internal/cache/cache.go

```diff
@@ -75,4 +79,10 @@ type Cache interface {
 
 	// SetExpTime adds expiration time of the pipeline to cache by pipelineId.
 	SetExpTime(ctx context.Context, pipelineId uuid.UUID, expTime time.Duration) error
+
+	// SetCatalog adds the given catalog to cache by ExamplesCatalog key.
+	SetCatalog(ctx context.Context, catalog []*pb.Categories) error
+
+	// GetCatalog return catalog from cache by ExamplesCatalog key.
+	GetCatalog(ctx context.Context) ([]*pb.Categories, error)
```

Comment:
Why can't we use already existing methods `GetValue` and `SetValue`? Is it because we use `Get`/`Set` for catalog instead of `HGet`/`HSet`?